Tests/BaseContainerTest.php

```diff
@@ -43,17 +43,17 @@ abstract class BaseContainerTest extends TestCase
     private $containerTestHandler;
 
     /**
-     * @var callable
+     * @var null|callable
      */
     private $createEventCallback;
 
     /**
-     * @var LoggersService
+     * @var null|LoggersService
      */
     private $logService;
 
     /**
-     * @var StorageApiService
+     * @var null|StorageApiService
```

Comment:
?StorageApiService ?

Reply:
PHPDoc AFAIK `?` jako nullable nezna, ne?

Reply:
Aspon v doc to nemaji teda https://docs.phpdoc.org/guide/guides/types.html

Reply:
No rozhodně mi to přijde lepší než k tomu připisovat |null :)